examples/models/fedper_cnn.py

```diff
@@ -1,31 +0,0 @@
-import torch
```

Comment:
Moved this model into the same file as that used for FedRep since they are both just sequentially split models but for different datasets (MNIST vs. CIFAR)